wsgi/odyssy/serve_static/templates/serve_static/base.html

```diff
@@ -0,0 +1,240 @@
+                                        <ul class="dropdown-menu">
+                                            <li><a href="/html/mtech_cse.html">CSE</a></li>
+                                        </ul>
+                                    </li>
+                                    <li><a href="/html/phd.html">Ph.D</a></li>
+                                    <li><a href="{% static 'basic/pdf/academic_calendar_autumn_2015_16.pdf' %}">Academic Calender</a></li>
+
+                                </ul>
+                            </li>
+                            <li>
+                                <a href="#" class="dropdown-toggle" data-toggle="dropdown">Admission <b
+                                        class="caret"></b></a>
+                                <ul class="dropdown-menu multi-level">
+                                    <li><a href="/html/btech_admission.html">B.Tech</a></li>
+                                    <li><a href="/html/mtech_admission.html">M.Tech</a></li>
+                                    <li><a href="/html/phd_admission.html">Ph.D</a></li>
+                                </ul>
+                            </li>
+                            <li>
+                                <a href="#" class="dropdown-toggle" data-toggle="dropdown">People <b class="caret"></b></a>
+                                <ul class="dropdown-menu multi-level">
+                                    <li><a href="/html/faculty.html">Faculty</a></li>
+                                    <li><a href="/html/visiting_faculty.html">Visiting Faculty</a></li>
+                                    <li><a href="/html/staff.html">Staff</a></li>
+                                </ul>
+                            </li>
+                            <li>
+                                <a href="#" class="dropdown-toggle" data-toggle="dropdown">More <b
+                                        class="caret"></b></a>
+                                <ul class="dropdown-menu multi-level">
+                                    <li><a href="/html/faq.html">FAQs</a></li>
+                                    <li><a href="/html/student_corner.html">Student's Corner</a></li>
+                                    <li><a href="http://moodle.iiitv.ac.in" target="_blank">Moodle</a></li>
+                                    <li><a href="/html/library.html">Library</a></li>
+                                    <li><a href="/html/invited_talks.html">Invited Talks</a></li>
+                                </ul>
+                            </li>
+                            <li><a href="/html/career.html">Career</a></li>
+                            <li><a href="/html/gallery.html">Photo Gallery</a></li>
+                        </ul>
+                    </div>
+                    <!--/.nav-collapse -->
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- /header -->
+    {% block body %}
```

Comment:
@singhpratyush There are two base.html files, server_static/base.html and templates/base.html .
Both are exactly same, there was some problem with linking the extra templates that's why this was done.
Also, the college logo and the students.jpg (tested for carousel) are required as they are a part of testing.